interp/constrintern.ml

```diff
@@ -572,10 +572,10 @@ let intern_assumption intern ntnvars env nal bk ty =
 
 let glob_local_binder_of_extended = DAst.with_loc_val (fun ?loc -> function
   | GLocalAssum (na,bk,t) -> (na,bk,None,t)
-  | GLocalDef (na,bk,c,Some t) -> (na,bk,Some c,t)
-  | GLocalDef (na,bk,c,None) ->
+  | GLocalDef (na,c,Some t) -> (na,Explicit,Some c,t)
```

Comment:
Is this somehow dropped?

Reply:
`bk` was always `Explicit` for `GLocalDef`. This moves the time we set the `bk` of a local def to a place where the structure does not allow us to drop it.

Told in another way, there is not enough structure in this quadruple to indicate that the `bk` is forced when the third argument is `Some`. (Maybe a sign that we should not use this quadruple anymore??)